core/record/node.js

```diff
@@ -192,6 +205,17 @@ const _getDatePart = (index) => R.pipe(R.partialRight(getValue, ['--']), R.split
 export const getDateYear = _getDatePart(0)
 export const getDateMonth = _getDatePart(1)
 export const getDateDay = _getDatePart(2)
+
```

Comment:
shouldnt these return numbers?

Reply:
yes, I'll change it and I'll check where they are used in the code.